lib/countUpperCase.js

```diff
@@ -4,8 +4,7 @@
  */
 
 function countUpperCase(str) {
-  // Your code goes here.
-  return;
+  return str.replace(/[^A-Z]/g, "").length;
```

Comment:
I have a tip for the future. If we are looking for capital letters to count then the easiest implementation to understand is one that does the same thing. I think of it as a direct solution. This one is sort of indirect in that it finds and removes everything that we aren't looking for.

Just a thought and a tip.